### DISCUSSION TITLE
How to achieve animated theme switch?

### DISCUSSION THREAD

**Discussion by luicfrr** [ant-design/ant-design#45175] [Q&A] [answered]
I want to achieve a theme switch just like docs site does. When click on theme switch button it'll have an animation switching to dark theme and animating back when switching to light theme. 

There's also an [issue](https://github.com/ant-design/ant-design/pull/44655) with this feature but I couldn't find anything on docs.

**Comment by yoyo837** [comment, marked as answer]
The animation effects mentioned in that issue are only experimental on the site. And `antd` components currently does not have it such a feature.

**Reply by luicfrr** [reply]
So this means there's no way for us to test this in our projects too?
